editor/editor_quick_open.cpp

```diff
@@ -33,6 +33,9 @@
 #include "core/os/keyboard.h"
 #include "editor/editor_node.h"
 #include "editor/editor_scale.h"
+#include "editor_settings.h"
```

Comment:
```suggestion
#include "editor/editor_settings.h"
```
The rest here have the full path, so use here as well